src/arrays/programs_involving_multidimensional_data/medium/q1/MainTest.java

```diff
@@ -0,0 +1,131 @@
+package arrays.programs_involving_multidimensional_data.medium.q1;
+
+import global.BaseTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.TestOption;
+import global.utils.ArrayUtil;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.PlaceHolder;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+public class MainTest extends BaseTest {
+    // Parsons with distractors
+
+    static final int size = 5;
+
+    public Clause[] testSentence() {
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = ArrayUtil.arrayToInput(ArrayUtil.generateRandomArray(1, 11, size * size * 2));
+        return clauseBuilder();
+    }
+
+    public void runMain() {
+        Identical.main(new String[0]);
+    }
+
+    static Stream<Arguments> inputProvider() {
+        int[][] a1 = new int[size][size];
+        int[][] b1 = new int[size][size];
+
```

Comment:
You should add a test case where the numbers aren't random. You could try doing something with ascending arrays or some other arrayUtil function

Reply:
that's what i was trying to do with
a3[x] = ArrayUtil.generateRandomArray(1, 2, size);
b3[x] = ArrayUtil.generateRandomArray(1, 2, size);
but after reviewing Kai's prs i remembered that the replicate method exists in array util so I can switch them to that